.github/workflows/run-e2e-tests.yml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
 
   push:
-    branches: [main]
+    branches: [main, fix-e2e-tests]
```

Comment:
can remove this and use the action UI to trigger manually as needed for testing

Reply:
was easier / automatic that way, but we can remove now